soccer/planning/planner/MotionCommand.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <variant>
+
+#include <Geometry2d/Point.hpp>
+#include <Geometry2d/Pose.hpp>
+#include <WorldState.hpp>
+#include "planning/trajectory/Trajectory.hpp"
+#include "planning/MotionInstant.hpp"
+#include "Utils.hpp"
+
+namespace Planning {
+//todo(Ethan) discuss whether we want to have settle with bounce
+struct SettleCommand{};
```

Comment:
I didn't include the option for robot.settle_w_bounce() in this rewrite because it seemed like it wasn't used. I'm not sure whether we were planning on using that soon. so I can add that back if needed.